src/components/NavBar.tsx

```diff
@@ -36,66 +53,110 @@ export class NavBar extends React.Component<INavBarProps, INavBarState> {
+            <em className="usa-logo-text">
+              <Link to="/" title="Digital VA home page">
+                <strong>VA</strong> | Developer Portal
+              </Link>
             </em>
           </div>
-          <button className="usa-menu-btn" onClick={this.toggleVisible}>Menu</button>
+          <button className="usa-menu-btn" onClick={this.toggleMenuVisible}>Menu</button>
         </div>
-        <nav className={this.setNavClass()} role="navigation">
+        <nav className={navClasses} role="navigation">
           <div className="usa-nav-inner">
-            <button className="usa-nav-close" onClick={this.toggleVisible}>
+            <button className="usa-nav-close" onClick={this.toggleMenuVisible}>
               <img src={closeButton} />
             </button>
+            <div className="usa-nav-secondary">
+              {apply}
+            </div>
             <ul className="usa-nav-primary usa-accordion">
-              <li>
-                <Link to ="/" className="usa-nav-link">
-                  <FontAwesomeIcon icon={faHome} />
-                  Home
-                </Link>
+              <li className="main-nav-item">
+                <MediaQuery query={LARGE_SCREEN_QUERY}>
+                  <NavLink to="/explore" className="usa-nav-link" activeClassName="default-nav-link" 
+                      isActive={this.checkActiveNavLink}>
+                    Documentation
+                  </NavLink>  
+                </MediaQuery>
+                <MediaQuery query={NOT_LARGE_SCREEN_QUERY}>
+                  <button className="usa-nav-link" onClick={this.toggleDocumentationSubMenu}>
+                    <span>Documentation</span>
+                    <img src={this.state.visibleSubNavs.documentation ? minusIcon : plusIcon} 
+                        alt="Expand Documentation" aria-label="Expand Documentation" className="nav-item-button" />
+                  </button>
+                </MediaQuery> 
+                {this.state.visibleSubNavs.documentation && this.renderDocumentationSubNav()}
               </li>
-              <li>
-                <Link to ="/explore" className="usa-nav-link">Documentation</Link>
-              </li>
-              <li>
-                <Link to ="/explore/benefits" className="usa-nav-link">Benefits</Link>
-              </li>
-              <li>
-                <Link to ="/explore/facilities" className="usa-nav-link">Facilities</Link>
-              </li>
-              <li>
-                <Link to ="/explore/health" className="usa-nav-link">Health</Link>
-              </li>
-              <li>
-                <Link to ="/oauth" className="usa-nav-link">OAuth</Link>
-              </li>
-              <li>
-                <Link to ="/explore/verification" className="usa-nav-link">Verification</Link>
+              <li className="main-nav-item">
+                <a href="mailto:api@va.gov" target="_blank" className="usa-nav-link"
```

Comment:
```suggestion
                <a href="mailto:api@va.gov" className="usa-nav-link"
```
The `target="_blank"` attribute here is going to cause a new tab to be opened before calling the default mail client. Removing this will fix that.

Reply:
is that the pattern we normally follow? if so, removing it

Reply:
Sorry, @mjlumetta I'm not familiar with if that's the pattern or not. I was just noting the issue and what a possible fix might be. For other sites though, I don't think that clicking an email link normally opens a new blank tab and then launches the default mail client.

Reply:
Agree with @jeff2d2. Let's remove.

Reply:
ohhh i see, sorry - i have mine set up to open in gmail, so it doesn't look like an issue for me. yeah, i see the benefit there.